sdk/keyvault/keyvault-keys/src/localCryptography/hash.browser.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { LocalCryptographyUnsupportedError } from "./models";
+
+/**
+ * @internal
+ * @ignore
+ * Use the platform-local hashing functionality
+ */
+export async function createHash(_algorithm: string, _data: Uint8Array): Promise<Buffer> {
```

Comment:
Although this is not realistically being used at the moment, it will be as soon as we get into browser support.